File: app/src/main/java/com/example/lucassong/clubsandwich/MainActivity.java
package com.example.lucassong.clubsandwich;

import android.content.Context;
import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button loginUserButton;
    private Button loginClubAdminButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        loginUserButton = findViewById(R.id.user_button);
        loginClubAdminButton = findViewById(R.id.club_admin_button);

        Context context = MainActivity.this;
        SharedPreferences sharedPref = context.getSharedPreferences(getString(R.string.username_pref_file_key), Context.MODE_PRIVATE);
        final SharedPreferences.Editor editor = sharedPref.edit();

        loginUserButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                editor.putString(getString(R.string.username_pref_file_key), "user");
                editor.commit();
                startActivity(new Intent(MainActivity.this, TimelineActivity.class));
            }
        });

        loginClubAdminButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                editor.putString(getString(R.string.username_pref_file_key), "admin");
                editor.commit();
                startActivity(new Intent(MainActivity.this, TimelineActivity.class));
            }
        });
    }
}

File: EXPERIMENTAL UI (NOT IN USE)/ClubSandwichNewsfeed/app/src/main/java/com/example/lucassong/clubsandwichnewsfeed/MainActivity.kt
package com.example.lucassong.clubsandwichnewsfeed

import android.os.Bundle
import android.support.design.widget.Snackbar
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import android.view.Menu
import android.view.MenuItem

import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.content_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)

        val posts: ArrayList<Post> = ArrayList()
        for (i in 0..100) {
            posts.add(Post("Title","Post number " + i,"http://cdn.playbuzz.com/cdn/925704be-9b8e-4dfc-852e-f24d720cb9c5/a6c64b24-9336-419c-a618-ea280cfb12c4_560_420.jpg"))
        }

        recyclerView.layoutManager = LinearLayoutManager(this)
        recyclerView.adapter = PostsAdapter(posts, this)
    }
}
